docs/versioned-plugins/inputs/beats-index.asciidoc

```diff
@@ -103,10 +103,6 @@ include::{include_path}/version-list-intro.asciidoc[]
 | <<v3.1.23-plugins-inputs-beats,v3.1.23>> | 2017-07-12
 | <<v3.1.22-plugins-inputs-beats,v3.1.22>> | 2017-07-12
 | <<v3.1.21-plugins-inputs-beats,v3.1.21>> | 2017-06-23
-| <<v3.1.20-plugins-inputs-beats,v3.1.20>> | 2017-06-23
-| <<v3.1.19-plugins-inputs-beats,v3.1.19>> | 2017-06-22
-| <<v3.1.18-plugins-inputs-beats,v3.1.18>> | 2017-06-07
-| <<v3.1.17-plugins-inputs-beats,v3.1.17>> | 2017-06-05
```

Comment:
We've seen this behavior before, and it's captured in this existing issue: https://github.com/elastic/docs-tools/issues/109